dbt/exceptions.py

```diff
@@ -130,3 +179,18 @@ def bad_package_spec(repo, spec, error_message):
     raise RuntimeException(
         "Error checking out spec='{}' for repo {}\n{}".format(
             spec, repo, error_message))
+
+
+def missing_config(model, name):
+    raise_compiler_error(
+        model,
+        "Model '{}' does not define a required config parameter '{}'."
+        .format(model.get('unique_id'), name))
+
+
+def invalid_materialization_argument(name, argument):
```

Comment:
is this used? I tried setting some extra config options but did not see this error